deploy/kubernetes/cloudability-metrics-agent.yaml

```diff
@@ -113,7 +113,7 @@ spec:
         - image: cloudability/metrics-agent:latest
           imagePullPolicy: Always
           securityContext:
-            runAsUser: 1000
+            runAsUser: 10000
```

Comment:
The dockerfile and readme still references user 1000 in multiple places

Reply:
Will update and test again

Reply:
Things look good after updating those locations